internal/provider/harness_docker_resource.go

```diff
@@ -154,14 +154,28 @@ func (r *HarnessDockerResource) Create(ctx context.Context, req resource.CreateR
 			return
 		}
 
+		var cpuResources *ContainerCpuResources
+		resp.Diagnostics.Append(resources.Cpu.As(ctx, &cpuResources, basetypes.ObjectAsOptions{})...)
+		if resp.Diagnostics.HasError() {
+			return
+		}
```

Comment:
I don't know enough about the tfsdk, will this just return a nil diagnostic if successful? So the conditional only triggers if `resources.Cpu.As` errors